### PR TITLE
release project popularity

### PR DESCRIPTION
**Pull request by klumb**
Signed-off-by: Kevin Lumbard <kmlumbard@gmail.com>

### CODE REVIEW

File: release/202102/metrics-value.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Identify the degree to which a project is valuable to its community of users (in
 <table>
   <thead><tr><th>Metric/Details</th><th>Question</th><th>Provide Feedback</th></tr></thead>
 <tbody>
+  <tr><td><a href="https://chaoss.community/metric-project-popularity">Project Popularity<br><img src="https://raw.githubusercontent.com/chaoss/website/master/release/Images/under-review100.png"></a></td><td>How does one measure the value of community interactions and accurately gauge “trust” within a community as evident from qualitative sentiment?</td><td><a href="https://github.com/chaoss/wg-value/issues/127">Issue 127</a></td></tr>
```
Comment (GeorgLink) [inline release/202102/metrics-value.md:14]: Fix the question:

```suggestion
  <tr><td><a href="https://chaoss.community/metric-project-popularity">Project Popularity<br><img src="https://raw.githubusercontent.com/chaoss/website/master/release/Images/under-review100.png"></a></td><td>How popular is an open source project?</td><td><a href="https://github.com/chaoss/wg-value/issues/127">Issue 127</a></td></tr>
```

**Review by GeorgLink — CHANGES_REQUESTED** [review]
LGTM

I suggest fixing the question.

**Comment by germonprez** [comment]
+1 on the question change. I though we solved this in the last meeting :)

**Comment by klumb** [comment]
My bad on the question, I copied from project velocity and forgot to edit that part.

**Comment by GeorgLink** [comment]
Thank you. No worries, we have the review process to catch stuff like this :)

**Pull request merged**